### ISSUE TITLE
rawwrite fails with "Error: LAYOUT: no sectors in sector definition!"

### ISSUE THREAD

**Issue by errolt** [davidgiven/fluxengine#787]
According to the usage.md page this should be acceptable parameters:

`fluxengine rawwrite -s <flux source> -d <flux destination>`

But 
`./fluxengine rawwrite -s test.flux -d test2.flux` results in:

> Error: LAYOUT: no sectors in sector definition!

Running a self compiled version that is up to,date with the repo.

**Comment by davidgiven** [comment]
Yes, the docs are out of date. You need to specify a format so that it knows which tracks you want to copy.

**Comment by davidgiven** [comment]
Reopening as a reminder to fix the documentation.
